### ISSUE TITLE
A11y_code.visualstudio.com_DownloadInsiders_ScreenReader: The table element is not provided with proper name nor it is provided with table headers.

### ISSUE THREAD

**Issue by MayuriNaik** [microsoft/vscode-docs#2796]
“[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Faccessibilityinsights.io%2F&data=02%7C01%7Cv-manai%40microsoft.com%7Cb67b2c4b646d4f9561a208d6f4b5c39b%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636965458847260936&sdata=T26HQfSGOlnuRQdX%2ByXk%2B2bxqgwFvCIVfuboZUWidYY%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**Environment Details:**
URL: https://code.visualstudio.com 
Windows Version: Windows10
Browser Version: 
Microsoft Edge 42.17134.1.0
Chrome Version 75.0.3770.100 (Official Build) (64-bit)
Screen Reader Versions : NVDA2019.1.1
MAS Violated : MAS 1.3.1, MAS 4.1.2, MAS 4.2.3

**Repro Steps:**
Navigate to the url: [https://code.visualstudio.com/insiders/](https://code.visualstudio.com/insiders/) 
follow below steps:
1.	Launch the application Url: [https://code.visualstudio.com](https://code.visualstudio.com) 
2.	Navigate to "Download" button.
3.	Scroll down the page and hit "Insiders build" link.
4.	Navigate to through the "Download Insiders" screen.
5.	Start the screen reader.
6.	Navigate to the table element in the main content and listen how it is announced.

**Actual:**
When screen reader users navigate through the table in the main content area consisting of the data such as User Installer, 64 bit, 32 bit. The table is not provided with a proper name to identify it and neither the table is provided with table headers to categorize the data within the table in certain category/group.

Similar issue was observed with "Edge + Narrator" as well as with "Chrome + NVDA".

**Expected:**
1. The table element should be provided with a proper name so as to know that the data within the table is regarding which content.
2. The table element should be provided with the row headers so as to understand that the data within the table is placed under with group.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
[https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx](https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx)

**User Impact:**
1. If the table element is not provided with proper name then the screen reader users might not know that the data within the table is for what.
2. If the table headers are not provided for the data table then the screen reader users will not know that the data in the table is under which category or group.

**MAS Reference:**
MAS 1.3.1 – [https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}) 

MAS 4.1.2 - [https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={248054a6-5e68-4771-9e1e-242fb5025730}](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={248054a6-5e68-4771-9e1e-242fb5025730}) 

MAS 4.2.3 - [https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={30e4fb40-dcc1-45b4-b714-5e2c28b23ae8}](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={30e4fb40-dcc1-45b4-b714-5e2c28b23ae8}) 

**Attachment for Reference:**
[Code.visualstudio.com_DownloadInsiders_ScreenReader Table has no name nor table headers..pptx](https://github.com/microsoft/vscode-docs/files/3320369/Code.visualstudio.com_DownloadInsiders_ScreenReader.Table.has.no.name.nor.table.headers.pptx)

**Comment by octref** [comment]
Tagged wrong fix.
